app/start/global.php

```diff
@@ -57,9 +57,13 @@
 });
 
 App::missing(function ($exception) {
-    return Response::view('errors.404', [
-        'pageTitle' => 'That page went missing!'
-    ], 404);
+    if (Request::is('dashboard*') or Request::is('api*')) {
+        return Response::view('errors.404', [
+            'pageTitle' => 'That page went missing!'
```

Comment:
That's not actually a psr-2 rule, but a rule from symfony's coding standards. I've got many of those fixers enabled on styleci because it enforces extra consistency, the same way I'm forcing short array syntax. psr-2 says nothing about that, and actually neither does symfony.